src/main/java/nl/tudelft/ti2206/group9/util/SaveGameParser.java

```diff
@@ -0,0 +1,81 @@
+package nl.tudelft.ti2206.group9.util;
+
+import nl.tudelft.ti2206.group9.level.State;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * @author Mathias
+ */
+public class SaveGameParser {
```

Comment:
*Auto-generated comment*

    Utility classes should not have a public or default constructor.